test/choosetests.jl

```diff
@@ -32,7 +32,8 @@ function choosetests(choices = [])
         "nullable", "meta", "profile", "libgit2", "docs", "markdown",
         "base64", "serialize", "functors", "misc", "threads",
         "enums", "cmdlineargs", "i18n", "workspace", "libdl", "int",
-        "checked", "intset", "floatfuncs", "compile", "parallel", "inline"
+        "checked", "intset", "floatfuncs", "compile", "parallel", "inline",
+        "boundscheck"
```

Comment:
`checkbounds` is already exported. Maybe it would be good to extend that namespace?